.github/workflows/reusable_receptorctl_tests.yml

```
@@ -0,0 +1,56 @@
+---
+name: receptorctl
+
+"on":
+  workflow_call:
+
+jobs:
+  receptorctl:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+        # NOTE: The highest and the lowest versions come
+        # NOTE: first as their statuses are most likely to
+        # NOTE: signal problems early:
+        - 3.11
+        - 3.8
+        - >-
+          3.10
+        - 3.9
+
+    env:
+      FORCE_COLOR: 1
+      NOXSESSION: tests-${{ matrix.python-version }}
+
+    name: >-  # can't use `env` in this context:
+      Run `receptorctl` tests-${{ matrix.python-version }} session
+
+    steps:
+    - name: Download the `receptor` binary
+      uses: actions/download-artifact@v4
+      with:
+        name: receptor
+        path: /usr/local/bin/
+
+    - name: Set executable bit on the `receptor` binary
+      run: sudo chmod a+x /usr/local/bin/receptor
+
+    - name: Set up nox
+      uses: wntrblm/nox@2024.04.15
+      with:
+        python-versions: ${{ matrix.python-version }}
+
+    - name: Check out the source code from Git
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
```

It just occurred to me that this might not be needed and I was about to suggest deleting this setting. Then I remembered there was something relying on Git in the noxfile... So this confusion with a setting needed in a place disconnected from the rest of the context calls for a code comment with a note:
```suggestion
        fetch-depth: 0  # Needed for the automation in Nox to find the last tag
```

*(copy it to the other workflow too?)*